packages/vuetify/src/composables/nested/nested.ts

```diff
@@ -209,6 +213,15 @@ export const useNested = (props: NestedProps) => {
         return arr
       }),
       register: (id, parentId, isGroup) => {
+        if (nodeIds.has(id)) {
+          const path = getPath(id).join(' -> ')
```

Comment:
`getPath(id)` can be a `Symbol`, which throws `TypeError: Cannot convert a Symbol value to a string`.

Reply:
2728a10244f4d9ae4268621e0cf310d8f29dab44